picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -1,908 +0,0 @@
-   * @see #load(File)
-   * @see #load(int)
-   */
-  @NonNull
-  public RequestCreator load(@Nullable String path) {
-    if (path == null) {
-      return new RequestCreator(this, null, 0);
-    }
-    if (path.trim().length() == 0) {
-      throw new IllegalArgumentException("Path must not be empty.");
-    }
-    return load(Uri.parse(path));
-  }
-
-  /**
-   * Start an image request using the specified image file. This is a convenience method for
-   * calling {@link #load(Uri)}.
-   * <p>
-   * Passing {@code null} as a {@code file} will not trigger any request but will set a
-   * placeholder, if one is specified.
-   * <p>
-   * Equivalent to calling {@link #load(Uri) load(Uri.fromFile(file))}.
-   *
-   * @see #load(Uri)
-   * @see #load(String)
-   * @see #load(int)
-   */
-  @NonNull
-  public RequestCreator load(@Nullable File file) {
-    if (file == null) {
-      return new RequestCreator(this, null, 0);
-    }
-    return load(Uri.fromFile(file));
-  }
-
-  /**
-   * Start an image request using the specified drawable resource ID.
-   *
-   * @see #load(Uri)
-   * @see #load(String)
-   * @see #load(File)
-   */
-  @NonNull
-  public RequestCreator load(@DrawableRes int resourceId) {
-    if (resourceId == 0) {
-      throw new IllegalArgumentException("Resource ID must not be zero.");
-    }
-    return new RequestCreator(this, null, resourceId);
-  }
-
-  /**
-   * Clear all the bitmaps from the memory cache.
-   */
-  public void evictAll() {
-    cache.clear();
-  }
-
-  /**
-   * Invalidate all memory cached images for the specified {@code uri}.
-   *
-   * @see #invalidate(String)
-   * @see #invalidate(File)
-   */
-  public void invalidate(@Nullable Uri uri) {
-    if (uri != null) {
-      cache.clearKeyUri(uri.toString());
-    }
-  }
-
-  /**
-   * Invalidate all memory cached images for the specified {@code path}. You can also pass a
-   * {@linkplain RequestCreator#stableKey stable key}.
-   *
-   * @see #invalidate(Uri)
-   * @see #invalidate(File)
-   */
-  public void invalidate(@Nullable String path) {
-    if (path != null) {
-      invalidate(Uri.parse(path));
-    }
-  }
-
-  /**
-   * Invalidate all memory cached images for the specified {@code file}.
-   *
-   * @see #invalidate(Uri)
-   * @see #invalidate(String)
-   */
-  public void invalidate(@NonNull File file) {
-    checkNotNull(file, "file == null");
-    invalidate(Uri.fromFile(file));
-  }
-
-  /** Toggle whether to display debug indicators on images. */
-  @SuppressWarnings("UnusedDeclaration") public void setIndicatorsEnabled(boolean enabled) {
-    indicatorsEnabled = enabled;
-  }
-
-  /** {@code true} if debug indicators should are displayed on images. */
-  @SuppressWarnings("UnusedDeclaration") public boolean getIndicatorsEnabled() {
-    return indicatorsEnabled;
-  }
-
-  /**
-   * Toggle whether debug logging is enabled.
-   * <p>
-   * <b>WARNING:</b> Enabling this will result in excessive object allocation. This should be only
```

Comment:
Don't see this documentation being migrated, might be useful.

Reply:
good catch!